api/src/test/java/org/openmrs/validator/AllergyValidatorTest.java

```diff
@@ -178,4 +179,18 @@ public void validate_shouldPassForAValidAllergy() {
 		
 		assertFalse(errors.hasErrors());
 	}
+
+	@Test
+	public void valid_shouldRejectNumericReactionValue() throws Exception {
+		Allergy allergy = new Allergy();
+		AllergyReaction reaction = new AllergyReaction();
+		String nonCoded = "sometext";
+		reaction.setReactionNonCoded(nonCoded);
+		allergy.addReaction(reaction);
+		Errors errors = new BindException(allergy, "allergy");
+		
```

Comment:
Do you need these extra spaces?

Reply:
Currently......can I remove them........indeed as in spaces